governance/voting.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Vote on a Proposal
+description: How to vote on a proposal so that it is enacted or rejected on Moonbeam via governance features
+---
+
+# Proposals
+
+![Governance Moonbeam Banner](/images/governance/governance-voting-banner.png)
+
+## Introduction
+
+Once a proposal reaches public referenda, token holders can vote on it using their own tokens. Two factors are defined by the weight a vote has: the number of tokens locked and lock duration (called conviction). This is to ensure that there is an economic buy-in to the result to prevent vote-selling. Consequently, the longer you are willing to lock your tokens, the stronger your vote will be weighted. You also have the option of not locking tokens at all, but vote weight is drastically reduced.
+
+Referenda are simple, inclusive, and stake-based voting schemes. Each referendum has a proposal associated with it that suggests an action to take place. They have a fixed duration, after which votes are tallied, and the action is enacted if the vote is approved.
+
+In Moonbeam, users will be able to create, second, and vote on proposals using their H160 address and private key, that is, their regular Ethereum address! 
```

Comment:
Should we link the section explaining the H160?

Reply:
I don't think is necessary because the next phrase says that is their regular Ethereum address. What I'll change is that Ethereum address for Ethereum account